neuralforecast/auto.py

```diff
@@ -1105,7 +1107,74 @@ def get_default_config(cls, h, backend, n_series=None):
 
         return config
 
-# %% ../nbs/models.ipynb 75
+# %% ../nbs/models.ipynb 74
+class AutoKAN(BaseAuto):
+
+    default_config = {
```

Comment:
Shouldn't this contain variations of the key parameters of the model, e.g. grid_size, spline_order?

Reply:
Good point! Fixed!